src/redux/books/books.js

```diff
@@ -0,0 +1,33 @@
+const ADD_BOOK = 'bookStore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+
+const initialState = [];
+
+const addBook = (payload) => ({
+  type: ADD_BOOK,
+  payload,
+});
+
+const removeBook = (payload) => ({
+  type: REMOVE_BOOK,
+  payload,
+});
+
+const reducer = (state = initialState, action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [
+        ...state,
+        action.payload,
+      ];
+    case REMOVE_BOOK: {
+      const newArray = [...state];
+      newArray.splice((action.payload), 1);
```

Comment:
- [ ] Please, to delete a book use the filter method like `state.filter(book => book.id !== id)`. Thisl will help to have a better delete function in your reducer. 🐶

Reply:
And easier too hahahah